schemas/zowe-yaml-schema.json

```diff
@@ -32,6 +32,17 @@
                   "type": "string",
                   "description": "Zowe PARMLIB"
                 },
+                "parmlibMembers": {
+                  "type": "object",
+                  "additionalProperties": false,
+                  "description": "Holds Zowe PARMLIB members",
+                  "properties": {
+                    "zis": {
+                      "type": "string",
+                      "description": "PARMLIB member for ZIS plugins"
```

Comment:
Needs better limits than just "string".

https://github.com/zowe/zowe-install-packaging/blob/v2.x/master/schemas/server-common.json#L20
Create an object here-ish, named $anchor=zoweDatasetMember
minlength=1, maxlength=8, pattern=alphanumerics
It's technically possible to have unprintable characters for member names, but it's quite crazy so let's just stick to alphanumerics.
Then back over here,
```
"properties": {
  "zis": {
    "$ref": "/schemas/v2/server-common#zoweDatasetMember",
    "description": "PARMLIB member used by ZIS"
  }
}
```

Reply:
Sure, makes sense!